fr/translation.json

```diff
@@ -235,7 +235,9 @@
   "contract_data": "Données du contrat",
   "bridge_txns": "Transactions du bridge",
   "code": "Code",
```

Comment:
@PaulUXD Missing `token_holders_transfers`

Reply:
oh will update now

Reply:
updated ready to review :)